### PR TITLE
frontend: bitbox02 refactoring views

### PR DESCRIPTION
**Pull request by thisconnect**

### CODE REVIEW

**Comment by thisconnect** [comment]
rebased

**Comment by thisconnect** [comment]
rebased

**Review by Beerosagos — APPROVED** [review]
tACK! Very nice, thanks @thisconnect :pray: 

I think the first commit has been already merged into master with a different hash, I think you can remove that. If you have time to polish a bit the commit messages would be nice to have them all separated imho

File: frontends/web/src/routes/device/bitbox02/setup/pairing.tsx [hunk 1]
```diff
@@ -25,32 +27,38 @@ type Props = {
...
+    getChannelHash(deviceID).then(({ hash, deviceVerified }) => {
+      setHash(hash);
+      setDeviceVerified(deviceVerified);
+    });
+  }, [deviceID]);
+
+  useEffect(onChannelHashChanged, [deviceID, onChannelHashChanged]);
+
+  useEffect(() => { // TODO: check useSubscribe instead
+    return channelHashChanged(deviceID, onChannelHashChanged);
```
[the remark this answers is not in the document]
Comment (thisconnect) [reply frontends/web/src/routes/device/bitbox02/setup/pairing.tsx:54]: added some comments

**Comment by thisconnect** [comment]
>  If you have time to polish a bit the commit messages would be nice to have them all separated imho

updated most commit messages and added a comment in the place where it was a bit unclear.

PTAL

File: frontends/web/src/routes/device/bitbox02/setup/pairing.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Beerosagos) [reply frontends/web/src/routes/device/bitbox02/setup/pairing.tsx:54]: Great, thanks

**Comment by Beerosagos** [comment]
> > If you have time to polish a bit the commit messages would be nice to have them all separated imho
> 
> updated most commit messages and added a comment in the place where it was a bit unclear.
> 
> PTAL

Looks good! Thanks

**Pull request merged**